### ISSUE TITLE
Unable to find suitable version for polymer

### ISSUE THREAD

**Issue by oneezy** [zacharytamas/page-title#1]
I'm using the Polymer Starter Kit

```
$ bower install page-title --save
bower page-title#*          not-cached https://github.com/zacharytamas/page-titl                       e.git#*
bower page-title#*             resolve https://github.com/zacharytamas/page-titl                       e.git#*
bower page-title#*            checkout v1.0.0
bower page-title#*            resolved https://github.com/zacharytamas/page-titl                       e.git#1.0.0
bower polymer#~1.0.7        not-cached git://github.com/Polymer/polymer.git#~1.0                       .7
bower polymer#~1.0.7           resolve git://github.com/Polymer/polymer.git#~1.0                       .7
bower polymer#~1.0.7          download https://github.com/Polymer/polymer/archiv                       e/v1.0.9.tar.gz
bower polymer#~1.0.7           extract archive.tar.gz
bower polymer#~1.0.7          resolved git://github.com/Polymer/polymer.git#1.0.                       9
bower polymer#^1.0.0            cached git://github.com/Polymer/polymer.git#1.4.                       0
bower polymer#^1.0.0          validate 1.4.0 against git://github.com/Polymer/po                       lymer.git#^1.0.0
bower polymer#^1.1.0            cached git://github.com/Polymer/polymer.git#1.4.                       0
bower polymer#^1.1.0          validate 1.4.0 against git://github.com/Polymer/po                       lymer.git#^1.1.0
bower polymer#^1.2.1            cached git://github.com/Polymer/polymer.git#1.4.                       0
bower polymer#^1.2.1          validate 1.4.0 against git://github.com/Polymer/po                       lymer.git#^1.2.1
bower polymer#^1.2.4            cached git://github.com/Polymer/polymer.git#1.4.                       0
bower polymer#^1.2.4          validate 1.4.0 against git://github.com/Polymer/po                       lymer.git#^1.2.4
bower                        ECONFLICT Unable to find suitable version for polym                       er

```

**Comment by oneezy** [comment]
I actually ran into this issue w/ another custom element "[Parallax Container](https://customelements.io/winhowes/parallax-container/)"

I haven't tested the fix out for your element yet, but one workaround was using "--force" for anyone else that comes across a similar issue.

`bower install page-title --save --force`

**Comment by oneezy** [comment]
Apparently all you have to do is  update your `bower.json` to version ^1.4.0 w/ the carrot and it will install properly for everyone.

`"dependencies": {
    "polymer": "^1.4.0"
    },`

**Comment by zacharytamas** [comment]
Thanks @oneezy. Yeah I need to go through some of my older elements and pin them to newer versions of Polymer. Polymer can be really complicated that way because everything is broken into so many separate Bower packages with interlinked dependencies. 😕

**Comment by oneezy** [comment]
yeah, i know what you mean! But I'm super excited about web components..just starting to get my feet wet w/ them.. but I'm liking what I'm seeing

**Comment by zacharytamas** [comment]
Oh yeah I totally love them. I've been using Polymer for almost two years on projects at Google and it's changed the way I think about front-end development. Really excited to see major Google products built with Polymer start going live.

**Comment by oneezy** [comment]
I've been reading about them for about 2 years..but I wanted to wait until 1.0 arrived before I really dove in. And man... me too! It's completely changing the way I do things. 

I'm making my current clients project completely w/ web components and SEO is an important factor w/ this one.. Glad you made this little `<page-title>` guy

**Comment by zacharytamas** [comment]
This is resolved in the 1.2.0 release.
